sigstore/_internal/tuf.py

```diff
@@ -144,6 +168,25 @@ def _setup(self) -> Updater:
 
         return updater
 
+    def _get_trusted_root(self) -> Optional[TrustedRoot]:
+        if not self._updater:
+            self._updater = self._setup()
+
+        root_info = self._updater.get_targetinfo("trusted_root.json")
+        if root_info is None:
+            return None
+        path = self._updater.find_cached_target(root_info)
+        if path is None:
+            try:
+                path = self._updater.download_target(root_info)
+            except (
+                TUFExceptions.DownloadError,
+                TUFExceptions.RepositoryError,
+            ) as e:
+                raise TUFError("Failed to download trusted key bundle") from e
+
+        return TrustedRoot().from_json(Path(path).read_bytes())
```

Comment:
We can skip it for now, but something to consider for a future refactor: it might be nicer/cleaner to use an `@lru_cache` decorated function here.